modules/node-observability-run-profiling-query.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/understanding-node-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="running-profiling-query_{context}"]
+= Running profiling query
+
+Profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. The Node Observability Operator stores the profiling data inside the container file system `/run/node-observability` directory. To request profiling data query, you have to create a `NodeObservabilityRun` resource.
+
+[IMPORTANT]
+====
+You can request only one profiling query at any point of time.
+====
+
+.Prerequisites
+* You have installed the Node Observability Operator.
+* You have created the `NodeObservability` custom resource (CR).
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `NodeObservabilityRun` resource file named `nodeobservabilityrun.yaml` that contains the following text:
```

Comment:
Sometimes in these commands you combine them with "cat <<EOF | oc apply -f -," and other times you don't. I don't have much of a preference, just bringing it to your attention.

Reply:
@stevsmit These procedures are tested by both QE and SME. I hope they are expecting it to be this way.